pkgs/by-name/mp/mphidflash/package.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  libusb-compat-0_1,
+}:
+
+stdenv.mkDerivation {
+  pname = "mphidflash";
+  version = "1.8-unstable-2020-05-05";
+
+  src = fetchFromGitHub {
+    owner = "AdamLaurie";
+    repo = "mphidflash";
+    rev = "d65bedd3f564a5d91b29e06b4a5885c32780acc5";
+    sha256 = "sha256-akQjkkbGxkurBifTZuI+iVs8O2i8MM0LgMUYztg5hzE=";
```

```suggestion
    hash = "sha256-akQjkkbGxkurBifTZuI+iVs8O2i8MM0LgMUYztg5hzE=";
```
use hash for sri hash